pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.version]
+path = "generic_parser/__init__.py"
+
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "/.github",
+  "/doc",
+  "/tests",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["generic_parser"]
+
+[project]
+name = "generic_parser"
+readme = "README.md"
+description = "An accelerator physics script collection for the OMC team at CERN."
+authors = [
+    {name = "PyLHC", email = "pylhc@github.com"},
+    {name = "Joschua Dilly", email = "joschua.dilly@cern.ch"},
+    {name = "Mael Le Garrec", email = "mael.le.garrec@cern.ch"},
+    {name = "Felix Soubelet", email = "felix.soubelet@cern.ch"},
+    {name = "Michael Hofer"},
```

Comment:
I don't really like that. I am getting enough spam as it is and I think github already takes track enough on who contributed.

Reply:
Thanks for noticing, I was thinking of pingin the relevant people about this but forgot after opening all these PRs. We can also just not provide email addresses apart from `pylhc@github.com`